### ISSUE TITLE
proxy option

### ISSUE THREAD

**Issue by jesus2099** [JonnyJD/musicbrainz-isrcsubmit#114]
Consider this as a **very low priority feature request**. :)

`--proxy http://coucou.toto.fr:80` would make all network streams go through the specified proxy.

**Comment by JonnyJD** [comment]
Yes, I was also considering this, since I am behind a firewall (forced using proxy) at my current workplace. Following the environment var `http_proxy` would also make sense.

However, I think this would first to be implemented at musicbrainzngs, because I am not making web requests myself (directly).

We are only talking proxies that are not transparent. Transparent ones should work out of the box.
However, there is also a bug using a transparent proxy on the MB server side:
http://tickets.musicbrainz.org/browse/MBS-6185

**Comment by jesus2099** [comment]
I should make a proxy feature request in [@alastair/python-musicbrainzngs/issues](/alastair/python-musicbrainzngs/issues)?
You have tagged this as upstream, but :thought_balloon: isn’t MB downstream for your isrcsubmitter (but I’m not an English expert)?
And oh I forgot about that `http_proxy` variable, I’ll try it…
I only require proxy when I access through VPN but otherwise proxy also works (without being necessary) but it slows down stuff…
Do you know what programs do use `http_proxy`? Only command line tools?
